src/elements/content-explorer/Content.js

```diff
@@ -64,9 +66,10 @@ const Content = ({
     tableRef,
     view,
     viewMode = VIEW_MODE_LIST,
+    metadataFieldsToShow = [],
```

Comment:
if the user doesn't pass in anything on the `metadataFieldsToShow` prop, it looks like the user will only see the empty state when in the metadata view. is this expected or are there any default fields we would show?

Reply:
yeah, empty state is ok because every metadata template is different.